matter/efr32/efr32mg24/BRD2601B/autogen/sl_component_catalog.h

```diff
@@ -10,18 +10,29 @@
 #define SL_CATALOG_BLUETOOTH_FEATURE_GATT_SERVER_PRESENT
 #define SL_CATALOG_BLUETOOTH_FEATURE_LEGACY_ADVERTISER_PRESENT
 #define SL_CATALOG_BLUETOOTH_FEATURE_NVM_PRESENT
-
 #define SL_CATALOG_BLUETOOTH_FEATURE_SM_PRESENT
 #define SL_CATALOG_BLUETOOTH_FEATURE_SYSTEM_PRESENT
+#define SL_CATALOG_BLUETOOTH_RTOS_ADAPTATION_PRESENT
 #define SL_CATALOG_BLUETOOTH_PRESENT
 #define SL_CATALOG_BLUETOOTH_FEATURE_MULTIPROTOCOL_PRESENT
 #define SL_CATALOG_CMSIS_OS_COMMON_PRESENT
+#define SL_CATALOG_DEVICE_INIT_PRESENT
+#define SL_CATALOG_DEVICE_INIT_CLOCKS_PRESENT
+#define SL_CATALOG_DEVICE_INIT_CORE_PRESENT
+#define SL_CATALOG_DEVICE_INIT_DCDC_PRESENT
+#define SL_CATALOG_DEVICE_INIT_DPLL_PRESENT
+#define SL_CATALOG_DEVICE_INIT_EMU_PRESENT
+#define SL_CATALOG_DEVICE_INIT_HFXO_PRESENT
+#define SL_CATALOG_DEVICE_INIT_LFRCO_PRESENT
+#define SL_CATALOG_DEVICE_INIT_LFXO_PRESENT
 #define SL_CATALOG_DEVICE_INIT_NVIC_PRESENT
 #define SL_CATALOG_EMLIB_CORE_PRESENT
 #define SL_CATALOG_EMLIB_CORE_DEBUG_CONFIG_PRESENT
 #define SL_CATALOG_FREERTOS_KERNEL_PRESENT
 #define SL_CATALOG_KERNEL_PRESENT
 #define SL_CATALOG_GATT_CONFIGURATION_PRESENT
+#define SL_CATALOG_HFXO_MANAGER_PRESENT
+#define SL_CATALOG_IOSTREAM_PRESENT
```

Comment:
I remember having an issue where we have this define elsewhere. Is that still the case?

Reply:
as discussed, This should be good now